server/controllers/templates/web_templates.go

```diff
@@ -114,6 +127,22 @@ var IndexTemplate = template.Must(template.New("index.html.tmpl").Parse(`
       <a href="{{ $basePath }}{{.LockPath}}">
         <div class="twelve columns button content lock-row">
         <div class="list-title">{{.RepoFullName}} <span class="heading-font-size">#{{.PullNum}}</span> <code>{{.Path}}</code> <code>{{.Workspace}}</code></div>
+        {{ if .Queue }}
```

Comment:
Any `web_templates` tests that we can add for these changes?

Reply:
Can't find any test regarding the rendered content

Reply:
Yep, I also could not find a test for this. I was hoping maybe we could add one in this PR to test verify/validate this change.